opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -197,10 +197,10 @@ message Profile {
   // If multiple binaries contribute to the Profile and no main
   // binary can be identified, mapping[0] has no special meaning.
   repeated Mapping mapping_table = 3;
-  // Locations referenced by samples via location_indices.
+  // Locations referenced by stacks via location_indices.
   repeated Location location_table = 4;
-  // Array of locations referenced by samples.
-  repeated int32 location_indices = 5;
+  // Stacks referenced by samples via stack_index.
+  repeated Stack stack_table = 5;
```

Comment:
The table should move to ProfilesData given that #644 was merged.

Reply:
Let's wait for #650 to get merged as it moves all lookup tables outside of `ProfilesData`.

Reply:
Done.